src/benchmarklib/benchmark_runner.cpp

```diff
@@ -190,7 +195,10 @@ void BenchmarkRunner::_create_report(std::ostream& stream) const {
     benchmarks.push_back(benchmark);
   }
 
-  nlohmann::json report{{"context", _context}, {"benchmarks", benchmarks}};
+  const auto total_run_duration_ns = std::chrono::duration_cast<std::chrono::nanoseconds>(_total_run_duration).count();
+  const auto total_run_duration_seconds = static_cast<float>(total_run_duration_ns) / 1'000'000'000;
```

Comment:
Couldn't we just use `std::chrono::duration_cast<std::chrono::seconds>` above instead of dividing here?

Reply:
Yeah, I guess we are fine with second-level granularity here.